libs/msgpack-c/Makefile

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (C) 2021 ImmortalWrt.org
+#
+# This is free software, licensed under the GNU General Public License v3.
```

Comment:
```suggestion
# SPDX-License-Identifier: GPL-2.0
```
Instead of the old style please use a SPDX tag and put it as very first line.

Reply:
Updated.